substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -96,7 +96,11 @@ pub struct StorageParams {
 	#[arg(long, default_value_t = 1)]
 	pub warmups: u32,
 
-	/// The `StateVersion` to use. Substrate `--dev` should use `V1` and Polkadot `V0`.
+	/// The `StateVersion` to use.
+	///
+	/// Substrate `--dev` should use `V1` and Polkadot `V0`.
+	/// To specify the state version, pass `--state-version=1` for `V1`
+	/// or `--state-version=0` for `V0`.
```

Comment:
Does this fix the `possible values: ` print from clap as well?

Reply:
`possible values: ` are auto-generated when using the clap enum, since we keep it as is, we won't have this messsage.